src/components/internal/DatePicker/DateRangePicker.tsx

```diff
@@ -13,18 +13,19 @@ export interface DateRangePickerProps {
   onSelect: (range: DateRange | undefined) => void;
   disabledDays?: Matcher | Matcher[];
   dottedDays?: Matcher[];
+  useYearPicker?: boolean;
```

Comment:
This makes me wonder if we should _always_ have this available? @jeffreyqua, what do you think about showing the Year navigation all of the time?

Reply:
@jeffreyqua ☝️